classes/03class/exercises/c03-docker06/ramiray/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine:3.12.0 AS base
+
+FROM base as src1
```

Comment:
Nice work. Just wondering about the purpose of this line though.

Reply:
Thanks Jay, 
Just trying to use the multi stage concept as per the exercise question. But not sure its correct or not . Waiting for the experts feedback .

Reply:
There is no point in use a multi-stage build in this case.
Multi-stage is useful when you do a lot of work in one container that will generate lots of files that are not useful for the final container image.
For example, if you need to create a container with a golang binary, you can have a multi-stage build where in the first container you download all libraries and compile the application. On the second container you just copy the binary generated from first container into the second one, so all libraries and code won't exist in the final container.

If this is not clear, slack em and I can try to explain you better.